modules/nf-core/learnmsa/align/meta.yml

```diff
@@ -0,0 +1,57 @@
+name: "learnmsa_align"
+description: Align sequences using learnMSA
+keywords:
+  - alignment
+  - MSA
+  - genomics
+tools:
+  - "learnmsa":
+      description: "learnMSA: Learning and Aligning large Protein Families"
+      homepage: "https://github.com/Gaius-Augustus/learnMSA"
+      documentation: "https://github.com/Gaius-Augustus/learnMSA"
+      tool_dev_url: "https://github.com/Gaius-Augustus/learnMSA"
+      doi: "10.1093/gigascience/giac104"
+      licence: "['MIT']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'test']`
+
+  - fasta:
+      type: file
+      description: Input sequences in FASTA format
+      pattern: "*.{fa,fasta}"
+
+  - meta2:
+      type: map
+      description: |
+        Groovy Map containing tree information
+        e.g. `[ id:'test_tree']`
+
+  - tree:
+      type: file
+      description: Input guide tree in Newick format
+      pattern: "*.{dnd}"
```

Comment:
I don't see that as an input in the main.nf

Reply:
You are right! Thanks for spotting this :)